front/lib/api/assistant/agent.ts

```diff
@@ -231,6 +232,15 @@ export async function* runMultiActionsAgentLoop(
 
           break;
 
+        case "agent_message_content":
+          // TODO(fontanierh): AgentMessage resource.
+          await AgentMessageContent.create({
```

Comment:
This is not very satisfying, but it wouldn't make sense having a `AgentMessageContentResource` , so removing this requires refactoring a `MessageResource` which is also a big effort.

Reply:
Can you provide more color here. Not sure I understand the comment?